pkg/txhandler/simple/policyloop.go

```diff
@@ -453,17 +453,17 @@ func (sth *simpleTransactionHandler) policyEngineAPIRequest(ctx context.Context,
 	}
 }
 
-func (sth *simpleTransactionHandler) HandleTransactionConfirmations(ctx context.Context, txID string, notification *apitypes.ConfirmationsNotification) (err error) {
+func (sth *simpleTransactionHandler) HandleTransactionConfirmations(ctx context.Context, tx *apitypes.ManagedTX, notification *apitypes.ConfirmationsNotification) (err error) {
```

Comment:
also updated the interface here to pass through the entire managed transactions so the other information (e.g. "from" field) becomes accessible.